modules/ossm-recommended-resources.adoc

```diff
@@ -4,11 +4,26 @@ This module included in the following assemblies:
 ////
 
 [id="ossm-recommended-resources_{context}"]
-= Recommended compute resources
+= Setting limits on compute resources
 
-By default, `spec.proxy` has the settings `cpu:10m` and  `memory:128M`.  If you are using Pilot, `spec.runtime.components.pilot` has the same default values.
-The settings in the following example are based on 1,000 services and 1,000 requests per second. You can change the values for `cpu` and `memory` as necessary based on your environment.
+By default, `spec.proxy` has the settings `cpu: 10m` and  `memory: 128M`.  If you are using Pilot, `spec.runtime.components.pilot` has the same default values.
 
+The settings in the following example are based on 1,000 services and 1,000 requests per second. You can change the values for `cpu` and `memory` in the `ServiceMeshControlPlane`.
+
+.Procedure
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. From the Project menu, select the project where you installed the control plane, for example `istio-system`.
```

Comment:
For future consideration: Service mesh has it's own control plane, apart from the OpenShift control plane? Should this be mentioned somewhere, or are your users smart enough to know the difference and which control plane you are referencing?

Reply:
I added a name for the resource to clarify.